tests/suite/manifests/snippets-filter/gateway.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: gateway
+spec:
+  gatewayClassName: nginx
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    hostname: "*.example.com"
+
```

Comment:
@lucacome do you know if anything changed such that the running of yamllint doesn't like having blank lines at the end of our files? I thought that this was the default we went off of?

Reply:
We should just have single newline at the end of the file.